lib/DDG/Spice/Emojipedia.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Emojipedia;
+
+# ABSTRACT: Provide definitions and meanings for emoji characters.
+
+use strict;
+use utf8;
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'http://api.emojipedia.org/emojis/$1?api_key={{ENV{DDG_SPICE_EMOJIPEDIA_APIKEY}}}';
```

Comment:
It also looks like there's some broken unicode coming through the JSON on your API?

![api_emojipedia_org_emojis_ __api_key_b43933e924b320e3df68ba38afa6a22f949d5a03_format_json](https://cloud.githubusercontent.com/assets/873785/14268683/dc77203a-faab-11e5-8e16-1598fba31c4b.png)